spec.bs

```diff
@@ -48,134 +50,162 @@ The following example shows a set of sensor values with [=Snippet=]s and afterwa
+All properties of datapoints are expandable to RDF, the RDF terms to use are defined through the path descriptions of a Snippet.
 
 <div class="example" id="ex_points">
 ```json
 [
-    { "time": "2024-01-01T06:00:00Z"^^xsd:dateTime, "value": "5.4"^^xsd:double, "id": "0"^^xsd:integer },
-    { "time": "2024-01-01T06:59:59Z"^^xsd:dateTime, "value": "5.2"^^xsd:double, "id": "1"^^xsd:integer },
-    { "time": "2024-01-01T08:00:00Z"^^xsd:dateTime, "value": "5.2"^^xsd:double, "id": "2"^^xsd:integer },
-    { "time": "2024-01-01T09:00:00Z"^^xsd:dateTime, "value": "6.1"^^xsd:double, "id": "3"^^xsd:integer },
+    { "time": "2026-01-01T06:00:00Z"^^xsd:dateTime, "value": "5.4"^^xsd:double, "id": "0"^^xsd:integer },
+    { "time": "2026-01-01T06:59:59Z"^^xsd:dateTime, "value": "5.2"^^xsd:double, "id": "1"^^xsd:integer },
+    { "time": "2026-01-01T08:00:00Z"^^xsd:dateTime, "value": "5.2"^^xsd:double, "id": "2"^^xsd:integer },
+    { "time": "2026-01-01T09:00:00Z"^^xsd:dateTime, "value": "6.1"^^xsd:double, "id": "3"^^xsd:integer },
 ]
 ```
 </div>
 
-## Extended Data Points ## {#extended-data-points}
-
-[=Data Point=]s can have extended values as well such as `cdt:Map` to allow complex values to be included.
-The RDF expansion is defined for each property in the `cdt:Map` by a `tss:Path`:
+Points can have extended values as well such as `cdt:Map` to allow complex values to be included:
 
 <div class="example" id="ex_snippet_advanced_ttl">
 ```turtle
-<snippet/2024-01-01>
+<snippet/2026-01-01>
   a tss:Snippet;
   tss:points "[
     { 
-      "time": "2024-01-01T06:00:00Z",
+      "time": "2026-01-01T06:00:00Z",
       "value": "{
         "latitude": "50.000000"^^xsd:double,
         "longitude": "4.000000"^^xsd:double,
         "temperature": "5.21"^^xsd:double
       }"^^cdt:Map
     },
     { 
-      "time": "2024-01-01T07:00:00Z",
+      "time": "2026-01-01T07:00:00Z",
       "value": "{
         "latitude": "50.100000"^^xsd:double,
         "longitude": "4.100000"^^xsd:double,
         "temperature": "7.61"^^xsd:double
       }"^^cdt:Map
     },
     { 
-      "time": "2024-01-01T08:00:00Z",
+      "time": "2026-01-01T08:00:00Z",
       "value": "{
         "latitude": "50.200000"^^xsd:double,
         "longitude": "4.200000"^^xsd:double,
         "temperature": "4.21"^^xsd:double
       }"^^cdt:Map
     }
   ]"^^cdt:List;
-  tss:from "2024-01-01T00:00.00Z"^^xsd:dateTime;
-  tss:until "2024-01-01T23:59:59Z"^^xsd:dateTime;
+  tss:from "2026-01-01T00:00.00Z"^^xsd:dateTime;
+  tss:until "2026-01-01T23:59:59Z"^^xsd:dateTime;
+  tss:pointType sosa:Observation;
+  tss:timePath sosa:resultTime;
+  tss:valuePath sosa:hasSimpleResult;
+  tss:about [
+    sosa:madeBySensor <temp_sensor_1>;
+    sosa:observedProperty <temperature>;
+  ];
+.
+```
+</div>
+
+Issue: this cdt:Map in a cdt:Map proposal is incompatible with paths! A possibility here is to allow the following:
```

Comment:
This 'issue' is solved with the `tss:Path` approach, if that's acceptable of course.

Reply:
I removed this again